### PR TITLE
Add stableswapPool.json and update docker-compose

### PR DESCRIPTION
**Pull request by phamminh0811**
Closes: #4239 

## What is the purpose of the change
Add stableswap pool to localosmosis's scripts folder for testing purposes.

## Brief Changelog
Add stableswapPool.json and update docker-compose

## Documentation and Release Note

  - Does this pull request introduce a new feature or user-facing behavior changes? (yes / no)
  - Is a relevant changelog entry added to the `Unreleased` section in `CHANGELOG.md`? (yes / no)
  - How is the feature or change documented? (not applicable   /   specification (`x/<module>/spec/`)  /  [Osmosis docs repo](https://github.com/osmosis-labs/docs)   /   not documented)

### CODE REVIEW

**Review by pysel — APPROVED** [review]
Thanks @phamminh0811!

File: tests/localosmosis/scripts/stableswapPool.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+	"initial-deposit": "1000000stake,1000uosmo",
+	"swap-fee": "0.005",
+	"exit-fee": "0",
+	"future-governor": "",
+    "scaling-factors": "1000,1"
```
Comment (ValarDragon) [inline tests/localosmosis/scripts/stableswapPool.json:6]: I think theres a whitespace error here

Comment (pysel) [reply tests/localosmosis/scripts/stableswapPool.json:6]: @phamminh0811

Comment (phamminh0811) [reply tests/localosmosis/scripts/stableswapPool.json:6]: i'm working on that

Comment (phamminh0811) [inline tests/localosmosis/scripts/stableswapPool.json:6]:
```suggestion
	"scaling-factors": "1000,1"
```

**Review by ValarDragon — APPROVED** [review]

**Pull request merged**